SimplenoteUITests/NoteList.swift

```diff
@@ -212,14 +231,27 @@ class NoteListAssert {
         NoteListAssert.noteListShown(forSelection: UID.NavBar.trash)
     }
 
+    class func noteContentIsShown(_ note: NoteData) {
+        noteContentIsShownInSearch(noteName: note.name, expectedContent: note.content)
+    }
+
     class func noteContentIsShownInSearch(noteName: String, expectedContent: String) {
-        print(">>> Asserting that note '\(noteName)' shows the following content:")
-        print(">>>> " + expectedContent)
+        print(">>> Asserting that note '\(noteName)' has the following content:")
+        print(">>>> \"\(expectedContent)\"")
+
+        guard NoteList.isNotePresent(noteName) else {
+            XCTFail(">>>> Note not found")
+            return
```

Comment:
I nice trick I like is to inline this

```suggestion
            return XCTFail(">>>> Note not found")
```

It works because the function returns `Void` and that exactly the return type of calling `XCTFail`.

Reply:
Oh nice! Thank you!